docs/src/rules/no-unused-vars.md

```diff
@@ -105,23 +107,45 @@ Note that `/* exported */` has no effect for any of the following:
 
 The line comment `// exported variableName` will not work as `exported` is not line-specific.
 
-Examples of **correct** code for `/* exported variableName */` operation:
+```js
+/* exported global_var */
+
+var global_var = 42;
+```
+
+Examples of **correct** code for `/* exported variableName */` operation with `no-unused-var`:
 
 ::: correct
 
 ```js
+/*eslint no-unused-vars: "error"*/
 /* exported global_var */
 
+// rule will not report global_var
+// sourcType must be set to script
+// environment should not be node or commonjs
+// globalReturn must be set to false
+
 var global_var = 42;
 ```
 
 :::
 
 ## Options
 
-This rule takes one argument which can be a string or an object. The string settings are the same as those of the `vars` property (explained below).
+This rule takes one argument which can be a string or an object.
 
-By default this rule is enabled with `all` option for variables and `after-used` for arguments.
+String setting has two possible modes `all` and `local` to allow global and local variables (works similar as `vars` option explained below).
+
+```json
+{
+    "rules": {
+        "no-unused-vars": ["error", "local"]
+    }
+}
+```
+
+Object setting has several options.
 
 ```json
 {
```

Comment:
I think this section was fine and seems better to leave it as is. The description of string settings refers to the `vars` property which is explained in detail below (note that `"local"` allows unused global variables, and neither of these two allows unused local variables, so the added description might be confusing). The config below is intended to show defaults for this rule, not an example of the object option.